### PR TITLE
[v7r3] docs: add info on installing non-released code from within the pilot

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*docs
NEW: add info on installing non-released code from within the pilot

ENDRELEASENOTES

### CODE REVIEW

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst [hunk 1]
```diff
@@ -68,3 +68,17 @@ DIRAC_JOB_PROCESSORS
...
 DIRAC_WHOLENODE
   A boolean flag indicating the pilot is exploiting the whole node (normally, False)
+
+
+Install non-released code from within the Pilot
+===============================================
+
+To install non-released code from within the pilot, the dirac-pilot command can be run with the --modules option. This option can be steered from the CS too.
+
+For example:
```
Comment (chrisburr) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst:78]: I think this should be?
```suggestion
For example::
```

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst [hunk 2]
```diff
@@ -68,3 +68,17 @@ DIRAC_JOB_PROCESSORS
...
+
+   dirac-pilot.py --modules=https://github.com/chaen/DIRAC.git:::DIRAC:::rel-v7r3_FEAT_proxyStrength
+
+would end up installing a specific branch (rel-v7r3_FEAT_proxyStrength) pushed to github.
```
Comment (chrisburr) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst:82]:
```suggestion
would end up installing a specific branch (``rel-v7r3_FEAT_proxyStrength``) pushed to github.
```

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst [hunk 3]
```diff
@@ -68,3 +68,17 @@ DIRAC_JOB_PROCESSORS
...
+
+An easy way to try is to in the CS configuration of one of your ComputingElements the option Modules=https://github.com/chaen/DIRAC.git:::DIRAC:::rel-v7r3_FEAT_proxyStrength
```
Comment (chrisburr) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst:84]:
```suggestion
An easy way to try is to in the CS configuration of one of your ComputingElements the option::

    Modules=https://github.com/chaen/DIRAC.git:::DIRAC:::rel-v7r3_FEAT_proxyStrength
```

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst [hunk 4]
```diff
@@ -68,3 +68,19 @@ DIRAC_JOB_PROCESSORS
...
+
+To install non-released code from within the pilot, the dirac-pilot command can be run with the --modules option. This option can be steered from the CS too.
+
+For example::
+
+   dirac-pilot.py --modules=https://github.com/chaen/DIRAC.git:::DIRAC:::rel-v7r3_FEAT_proxyStrength
+
+would end up installing a specific branch (``rel-v7r3_FEAT_proxyStrength``) pushed to github.
+
+An easy way to try is to in the CS configuration of one of your ComputingElements the option::
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst:84]:
```suggestion
An easy way to try is to add the Modules option in the CS configuration of one of your ComputingElements::
```
Phrase was missing "add" somewhere

**Comment by andresailer** [comment]
I was looking if the options available for ComputingElements are documented somewhere in one place, but I don't think they are. But that is something for a different PR.

PS: this `:` is a colon, this `;` is a semicolon

**Review by andresailer — APPROVED** [review]

**Pull request merged**